Independant Register View

Lailouezzz opened the pull request:
Added: RegisterView
Modified: LoginView now have only 1 button LOGIN
Modified: HeaderBar now have 2 button (login and register)
Modified: New route for registerview

Dyamen1411 committed:
Api request would not format payload correctly.

Fix:
	- If you tried to post using
		`makeApiQuery`, the payload would
		not get passed properly.

Dyamen1411 reviewed (approved):
Connected view to the back. Registration is now possible.

Dyamen1411 committed:
Made registration possible.

Lailouezzz committed:
Added: register view
Modified: LoginView now have only 1 button LOGIN
Modified: HeaderBar now have 2 button (login and register)
Modified: New route for registerview
